cypress/integration/login.js

```diff
@@ -11,14 +11,15 @@ describe('Login functionality', function() {
         cy.get('[id="loginForm.userId"]').click({force:true})
         cy.get('[aria-label="Demo User"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.main-nav').find('li').should('have.length', 2)
+        cy.get('[data-test="timesheets-current-day"]').contains(15);
```

Comment:
This test works great today, but it will no longer work tomorrow, because the date '15' is hardcoded. It would be great to use a JS method like getDate() to retrieve the day of month whenever. 

Also, there seems to be a bug in our system where every calendar cell contains the same `[data-test="timesheets-current-day"]` element. Let's change it to `cy.get('.calendar--today')`, as this class only appears on today's cell :)